test/FileService.Mappers.UnitTests/ImageMapperTests.cs

```diff
@@ -73,7 +73,7 @@ public class ImageMapperTests
         private FakeHttpContextAccessor fakeHttpContextAccessor;
 
         private ImageRequest imageRequest;
-        private byte[] resizedImageContent;
+        private string resizedImageContent;
```

Comment:
resolve warnings 
![image](https://user-images.githubusercontent.com/38541098/113492429-ef662a00-94df-11eb-87aa-d67eb43dd579.png)

Reply:
I think this is not my task